less/CreditCard.less

```diff
@@ -0,0 +1,25 @@
+.credit-card-update--register-wrapper {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.credit-card-update--register-terminal {
+    width: 99%;
+    height: 311px;
```

Comment:
Probably needs more height as, at least for me, the iframe doesn't fit in `311px`.
Also, should we remove borders from this (`border: 0`)?

Reply:
I tried removing  borders but it makes the right side disappear while the others are still there (I guess it's out of our control).

> Probably needs more height as, at least for me, the iframe doesn't fit in 311px

I wonder what might be the reason for this, because in my screen things are fine. Isn't there any relative way to specify height? 🤔  Maybe not since we don't control the content of the iframe.

Reply:
> Maybe not since we don't control the content of the iframe

This is indeed the case

Reply:
In vetrina we have the following styles for the iframe:
```
.vetrina--payment-terminal {
    background: #fff;
    width: 550px;
    height: 360px;
    border: none;
}
```

Reply:
Ok let's think about this in another issue, since I can't seem to find a nice way to display the iframe in the account edit box.